recipe/meta.yaml

```diff
@@ -15,7 +15,6 @@ source:
 
 build:
   number: 0
-  skip: True  # [win]
```

Comment:
Ah. I missed that! @scopatz is correct! You do need to re-render. (Sorry for the confusion.)